test/json_api_files/set_active_track_radius_meters.json

```diff
@@ -0,0 +1 @@
+{"setActiveTrack":{"radius":7,"track":{"id":8888,"type":0,"sf":[1.0,1.0]}}}
```

Comment:
suggest a test case to verify setting an active track that also includes sector points.